src/controllers/googleFacebook.js

```diff
@@ -22,7 +22,8 @@ export const userToken = async (req, res) => {
   };
 
   const token = await storeToken(user);
-  if (req.user.socialAuth) return res.redirect(`${process.env.BASE_URL}?accessToken=${token.accessToken}&refreshToken=${token.refreshToken}`);
+  // if (req.user.socialAuth) return res.redirect(`${process.env.FRONTEND_URL}?auth/${user.id}`);
+  if (req.user.socialAuth) return res.redirect(`${process.env.FRONTEND_URL}?accessToken=${token.accessToken}&refreshToken=${token.refreshToken}`);
```

Comment:
remove the tokens from the link. chain it as `res.redirect(FrontendURL).send(tokens)

If this works then on the frontend the tokens should be picked from the response object not the url